tests/sentry/lang/javascript/test_processor.py

```diff
@@ -169,6 +170,15 @@ def test_unicode_body(self):
 
         assert result == result2
 
+    @responses.activate
+    def test_truncated(self):
+        url = truncatechars('http://example.com', 3)
```

Comment:
This test helps explicitly test this behavior. Just in case, say we started truncating with a different character like `…` instead of `...`, this test will trigger a failure.